collect_app/src/main/java/org/odk/collect/android/preferences/GeneralPreferencesFragment.java

```diff
@@ -64,31 +63,36 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
 
     @Override
     public boolean onPreferenceClick(Preference preference) {
-        Fragment fragment = null;
         switch (preference.getKey()) {
             case "protocol":
-                fragment = new ServerPreferences();
+                getActivity().getFragmentManager()
```

Comment:
Can those similar code be extracted into a method? And put an instance as parameter.

Reply:
I have fixed the code redundancy to match the way it was done in the original code